imapautofiler/tests/base.py

```diff
@@ -81,3 +87,11 @@ def i18n_msg(self):
                 construct_message(I18N_MESSAGE)
             )
         return self._msg
+
+    @property
+    def recent_msg(self):
+        if self._msg is None:
```

Comment:
We should use a different instance variable to hold this property in case a test wants to access both message objects. Maybe self._recent_msg instead of self._msg?